### ISSUE TITLE
WebApp:Can't remove Deny access-restriction rule

### ISSUE THREAD

**Issue by kmadof** [Azure/azure-cli#15409]
**Describe the bug**
I can't remove rule when I have only Deny rules in access restriction.

**To Reproduce**
Add `Deny` rule
` az webapp config access-restriction add -g TheCodeManual -n Hadar --slot Slot-test --rule-name block-me --action Deny --ip-address 79.184.1.1/1 --priority 199`

Try remove it:
`az webapp config access-restriction remove -g TheCodeManual -n hadar --slot Slot-test --rule-name block-me`

And you will get:
> ValidationError: No rule found with the specified criteria

**Expected behavior**
I should be able to remove rules regardless of their type.

**Environment summary**
az --version
```
azure-cli                         2.12.0 *

core                              2.12.0 *
telemetry                          1.0.6

Extensions:
azure-devops                      0.18.0

Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Extensions directory 'C:\Users\krzys\.azure\cliextensions'

Python (Windows) 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 23 2018, 23:31:17) [MSC v.1916 32 bit (Intel)]

Legal docs and information: aka.ms/AzureCliLegal
```
$PSVersionTable.PSVersion
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      18362  752
```
`[System.Environment]::OSVersion.Version`
```
Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      18363  0
```

**Comment by yungezz** [comment]
routing to appropriate team

**Comment by btardif** [comment]
@Kotasudhakarreddy I think you fixed this already, can you validate and close?

**Comment by madsd** [comment]
@kmadof , @btardif This works as expected. To remove a deny rule, you currently have to specify explicitly --action Deny as you could have two rules with the same name respectively allowing and denying separate conditions. We can update the validation error to: ValidationError: No rule found with the specified criteria. If you are trying to remove a Deny rule, you must explicitly specify --action Deny

**Comment by Kotasudhakarreddy** [comment]
@madsd - Thanks for your feedback. Will check and change the validation error as more informative.

**Comment by kmadof** [comment]
@madsd Thanks for replying. I will check that today!

**Comment by btardif** [comment]
@panchagnula I talked to @madsd and he was going to help take a first pass with all the access restrictions related items.

**Comment by madsd** [comment]
@kmadof Did you have a chance to verify?

**Comment by btardif** [comment]
This is fixed by #15784
